docs/docs/how-to/s3-custom-endpoint.md

```diff
@@ -0,0 +1,18 @@
+---
+id: s3-custom-endpoints
+title: Use custom endpoint for s3 hosts
+---
+
+In this how-to guide, we will see how we can connect to differrent **S3 Hosts** using the custom endpoint. In this guide, we using minio since it is an S3 compatible object storage. 
```

Comment:
```suggestion
In this how-to guide, we will see how we can connect to differrent **S3 Hosts** using the custom endpoint. In this guide, we are using Minio since it is an S3-compatible object storage. 
```